Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -92,7 +141,8 @@ def test_all_incidents_in_helios(self):
         print("Starting test_all_incidents_in_helios")
         ids = get_incident_ids(self.resource_group, self.workspace_name)
         alert_ids = [alert_id for (incident_id, alert_id) in ids]
-        for alert_id in alert_ids:
+        pattern = r"^\d+:\d+$"
```

Comment:
Please add a comment about why we're doing this.

Actually, please add a validation that there's exactly one such message and it's title has 'Hello From Cohesity' in it.

Reply:
done